internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -184,8 +184,14 @@ func verifyInputs(spec *loggingv1.ClusterLogForwarderSpec, status *loggingv1.Clu
 		validPort := func(port int32) bool {
 			return port == 0 || (port > 1023 && port < 65536)
```

Comment:
Now with Port being required in the ReceiverSpec in the API and without HTTPReceiver.GetPort(), we need to disallow port 0 as a valid port. In the prior version, it was here as a placeholder to be substituted for the default 8443, but it doesn't make sense anymore. Deferring to @alanconway as an API concern.

Reply:
This is a regression. I don't understand why we have to require the port.  Why can we not provide a validation that spins through the listeners and ensures they are unique from all other ports including the ones we reserve for metrics

Reply:
My new update leaves the Port not required.